notebooks/datasets/nldas_compare2_3.ipynb

```diff
@@ -0,0 +1,691 @@
+{
```

Comment:
<p>Comparing the Data should be a new chapter</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NASA-IMPACT/veda-docs/pull/154/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>